README.md

```diff
@@ -40,3 +40,22 @@ smaiyya@tem121:~/sgx/adrian/ORTOA/build/ $ ortoa-simulate -h
 ```bash
 smaiyya@tem121:~/sgx/adrian/ORTOA/build/ $ ortoa-client-run -h
 ```
+
+
+# Previously in `dependencies.txt`
```

Comment:
TODO: Write this out nicely in the README. Possibly do some stuff with cmake to pull these automatically